doc_source/programming-client.md

```diff
@@ -55,4 +55,9 @@ $KmsClient = new Aws\Kms\KmsClient([
 ]);
 ```
 
-------
\ No newline at end of file
+------
+#### [ Node.js ]
+
+```js
+const kms = new AWS.KMS({ apiVersion: '2014-11-01' });
```

Comment:
The `apiVersion` is optional.  I would not use it here.

Reply:
ok done